src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+of the duration of the `keydown` and the `keyup` events. The `keypress` event is deprecated, and it
+should be fired in the same task as the `keydown` event, so its duration will always overlap with
+the duration of the`keydown`.
+
+{% Details %}
+
+{% DetailsSummary %}
+A note on repeating keypresses
+{% endDetailsSummary %}
+
+There is an edge case here worth mentioning: there may be cases where the user presses a key and
+takes a while to release it. In this case, the sequence of events dispatched can
+[vary](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#auto-repeat_handling). In
+these cases, we would consider there to be one interaction per `keydown`, which may or may not have
+a corresponding `keyup`. {% endDetails %}
+
+##### Tap
+
+Another important user interaction is when the user taps or clicks on a website. Similar to
+`keypress`, some events are fired as the user presses down, and others as they release, as shown in
+the diagram above, Note the events associated with a tap are a little different on desktop vs
+mobile.
+
+For a tap or click, the release is generally the one which triggers the majority of reactions, but,
+as with keyboard interactions, we want to capture the full interaction. And in this case it’s more
+important to do so because having some UI updates upon tap press is not actually that uncommon.
+
+We’d like to include the event durations for all of these events, but as many of them overlap
+completely, we need to measure just `pointerdown`, `pointerup`, and `click` to cover the full
+interaction.
+
+{% Details %}
+
+{% DetailsSummary %}
+Can we narrow further to just `pointerdown` and `pointerup`?
+{% endDetailsSummary %}
+
+One initial thought would be to use the `pointerdown` and `pointerup` events and assume that they
+cover all of the durations that we’re interested in. Sadly, this is not the case, as this [edge
+case](https://output.jsbin.com/buyiyew/quiet) shows. Try opening this site on mobile, or with mobile
+emulation, and tapping where it says “Click me”. This site triggers the [browser tap
+delay](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away). It can be seen
+that the `pointerdown`, `pointerup`, and `touchend` are dispatched quickly, whereas the `mousedown`,
+`mouseup`, and `click` wait for the delay before being dispatched. This means that if we only looked
+at `pointerdown` and `pointerup` then we’d miss the duration from the synthetic events, which is
+large due to the browser tap delay and should be included. So we should measure `pointerdown`,
+`pointerup`, and `click` to cover the full interaction. {% endDetails %}
+
+##### Drag
+
+We decided to include dragging as well since it has similar associated events and since it generally
+causes important UI updates to sites. But for our metric we intend to only consider the drag start
+and the drop - the initial and final parts of the drag. This is to make it easier to reason about as
```

Comment:
Nit: do all drags lead to "drops"?  I think many drags are custom scrollers?  Perhaps just call it `drag start and stop`?